conduce.py

```diff
@@ -22,6 +23,25 @@ def list_from_args(args):
     return api.list_resources(object_to_list.upper().rstrip('S'), **vars(args))
 
 
+def create_resource(args):
+    resource_type = args.type.upper()
+    resource_name = args.name
+    content_path = args.content
+
+    del vars(args)['name']
+    del vars(args)['type']
+    del vars(args)['content']
+
+    with open(content_path, 'rb') as content_stream:
+        if resource_type.lower() == 'asset':
```

Comment:
nitpick, but if you're in here, it seems weird to determine the mime type right after opening the file, even though it's not using anything from the file content.  I'd move this before the `with`

Reply:
I thought about that.  The reasoning is that there is no point in determining the mime type if the file will not open.